### PR TITLE
[React] Unused code

### PR DESCRIPTION
**Pull request by qmonmert**
Remove unused code

---

Please make sure the below checklist is followed for Pull Requests.

- [x] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
- [x] Tests are added where necessary
- [x] [jhipster-online](https://github.com/jhipster/jhipster-online) was updated if necessary
- [x] Documentation is added/updated where necessary
- [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (bellow reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

File: generators/client/templates/react/src/main/webapp/app/modules/administration/administration.reducer.spec.ts.ejs [hunk 1]
```diff
@@ -41,7 +41,6 @@ import administration, {
 } from './administration.reducer';
 
 describe('Administration reducer tests', () => {
-  const username = process.env.E2E_USERNAME ?? 'admin';
```
Comment (mshima) [inline generators/client/templates/react/src/main/webapp/app/modules/administration/administration.reducer.spec.ts.ejs:44]: Seems to be used for websocket tests.

Comment (qmonmert) [reply generators/client/templates/react/src/main/webapp/app/modules/administration/administration.reducer.spec.ts.ejs:44]: you're right, I fix that

Comment (mshima) [reply generators/client/templates/react/src/main/webapp/app/modules/administration/administration.reducer.spec.ts.ejs:44]: Move to line 229, can remove the condition

Comment (qmonmert) [reply generators/client/templates/react/src/main/webapp/app/modules/administration/administration.reducer.spec.ts.ejs:44]: yes

Comment (mshima) [reply generators/client/templates/react/src/main/webapp/app/modules/administration/administration.reducer.spec.ts.ejs:44]: Thanks.

**Review by mshima — APPROVED** [review]

**Pull request merged**